taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      </p>
+      <h4 id="storage_data_path">
+        Determining the Path to Your Transfer Directory
+      </h4>
+
+      <p>
+        A transfer directory on the remote storage server has already been set up for you and is available for data transfer. The transfer directory path is individualized to your research center/institution.
+      </p>
+      <details class="small"><summary>Examples:</summary>
+        <pre><code>/corral-secure/projects/A2CPS/submissions/utaustin/</code></pre>
+        <blockquote>
+          <p>
+            Note: The transfer directory name will always be one word and in lowercase.
+          </p>
+        </blockquote>
+      </details>
+      <p>
+        If you are unsure of your transfer directory path, please consult the email you have received from a relevant contact <samp>@tacc.utexas.edu</samp>.
+      </p>
+      <h4 id="scp_transfer">Transfer Data with<code>scp</code></h4>
+
+      <p>
+        <code>scp</code>copies files between hosts on a network. To transfer a file (ex. <code>my_file.txt</code>) to the remote secure system via <code>scp</code>, open a terminal on your local computer and navigate to the path where your data file is located.
+      </p>
+      <pre style="max-width: 500px"><code><u>localhost$ </u>cd Documents/portal-data/</code></pre>
+
+      <p>
+        Assuming your TACC username is <code>jdoe</code> and you are affiliated with UT Austin, a <code>scp</code> transfer that pushes <code>my_file.txt</code> from your local computer to the remote secure system would look like this:
+      </p>
+      <pre style="max-width: 1500px"><code><u>localhost$ </u>scp ./my_file.txt jdoe@<kbd>host</kbd>:<kbd>/transfer/directory/path</kbd></code></pre>
+      <blockquote>
+        <p>
+          Note: This command will copy your data file directly to your individualized transfer directory on the remote storage system.
+        </p>
+      </blockquote>
+      <p>
+        <em>If you have not done so already, enter this command in your terminal,
+          replacing the file name, TACC username, and your individualized transfer directory name appropriately.</em>
+      </p>
+      <p>
+        After entering the command, you will be prompted to login to the remote secure system by entering the password associated with your TACC account as well as the token value generated from your TACC token app.
+      </p>
+      <p>
+        The data transfer was successful if your terminal output looks like this:
+      </p>
+      <pre style="max-width: 500px"><samp>my_file.txt     100% ##  #.#          KB/s   ##:##</samp></pre>
+
+      <p>
+        If you wish to learn more about <code>scp</code> and how to synchronize your file transfer, you can do so
+        <a target="_blank" href="https://man7.org/linux/man-pages/man1/scp.1.html">the online <code>man</code> page for <code>scp</code></a>
+        or follow the file transfer section of the user guide for the relevant storage system:
```

Comment:
```suggestion
        or follow the file transfer section of the user guide for the appropriate TACC system:
```

@jgentle Should this instance also change (like [another instance that you suggested we change](https://github.com/TACC/Core-CMS/pull/152/files/89cd2a51d3b0904421df9438be2385f46c0f7ae6?file-filters%5B%5D=.html#r597301854))?

Reply:
For consistency sake across the guides I would say yes. Good catch!